plugins/modules/mount.py

```diff
@@ -93,7 +93,10 @@
   boot:
     description:
       - Determines if the filesystem should be mounted on boot.
-      - Only applies to Solaris systems.
+      - Only applies to Solaris and Linux systems.
```

Comment:
Hi @saito-hideki why only on Solaris and Linux ? Can't it be extended to *BSD too ? Regarding fstab format, Solaris is the only one system that differs from all others. I guess the `noauto` option is very portable, no ?

Reply:
Yeah that make sense, I'm planning that once this PR will be merged, I'll expand it for *BSD :)